contracts/BConst.sol

```diff
@@ -16,6 +16,8 @@ pragma solidity 0.5.12;
 import "./BColor.sol";
 
 contract BConst is BBronze {
+    address private SLOT_FOR_PROXY;
```

Comment:
what is this var for?

Reply:
The proxy takes up the first slot of storage, this is needed so it doesn't overwrite another slot, maybe it wont be needed with OZ clone though.

Reply:
Removed with the help of Clones, thanks